processor/tailsamplingprocessor/processor_test.go

```diff
@@ -66,49 +65,55 @@ func (t *TestPolicyEvaluator) Evaluate(ctx context.Context, traceID pcommon.Trac
 // testTSPController is a set of mechanisms to make the TSP do predictable
 // things in tests.
 type testTSPController struct {
-	// tickBarrier is a barrier to block ticks until the test is ready
-	tickBarrier chan struct{}
+	tickChans []chan chan struct{}
```

Comment:
I think this can just be a `chan chan struct` now. The slice was for having multiple workers